test/async_scope/test_stop.cpp

```diff
@@ -6,18 +6,18 @@ namespace ex = std::execution;
 using _P2519::execution::async_scope;
 using _P2300::this_thread::sync_wait;
 
-TEST_CASE("TODO: calling request_stop will cancel the async_scope object", "[async_scope][stop]") {
-  async_scope scope;
-
-  scope.request_stop();
-
-  // TODO: reenable this
-  // REQUIRE(P2519::__scope::empty(scope));
-  impulse_scheduler sch;
-  scope.spawn(ex::on(sch, ex::just()));
-  // TODO: reenable this
-  // REQUIRE(P2519::__scope::empty(scope));
-}
+// TEST_CASE("TODO: calling request_stop will cancel the async_scope object", "[async_scope][stop]") {
+//   async_scope scope;
+
+//   scope.request_stop();
+
+//   // TODO: reenable this
+//   // REQUIRE(P2519::__scope::empty(scope));
+//   impulse_scheduler sch;
+//   scope.spawn(ex::on(sch, ex::just()));
+//   // TODO: reenable this
+//   // REQUIRE(P2519::__scope::empty(scope));
+// }
```

Comment:
Why is this test case disabled?

Reply:
@lucteo wrote these and disabled them - I would like to have him take a look at these tests